plugin/adsk/plugin/importTranslator.cpp

```diff
@@ -113,7 +113,10 @@ UsdMayaImportTranslator::reader(
             userArgs,
             /* importWithProxyShapes = */ false,
             timeInterval);
-    UsdMaya_ReadJob mUsdReadJob(fileName, primPath, variants, jobArgs);
+    UsdMaya_ReadJob mUsdReadJob(
+        fileName,
+        MayaUsd::ImportData::instance(),
```

Comment:
Seems like we should be constructing an ImportData from the arguments rather than pulling from a singleton.